src/main/java/seedu/address/model/Model.java

```diff
@@ -192,4 +192,10 @@
      * Add an user-defined alias to the current mapping.
      */
     void addAlias(Alias alias);
+
+    /**
+     * Deletes an user-defined alias from the current mapping.
+     * @param aliasName The name of the alias to be deleted.
```

Comment:
Should there be a blank line between description and parameters?

Reply:
Done!